client/me/help/help-search/index.jsx

```diff
@@ -12,40 +11,30 @@ import { localize } from 'i18n-calypso';
 /**
  * Internal dependencies
  */
-import HelpSearchStore from 'lib/help-search/store';
-import HelpSearchActions from 'lib/help-search/actions';
+import CompactCard from 'components/card/compact';
+import getHelpLinks from 'state/selectors/get-help-links';
 import HelpResults from 'me/help/help-results';
 import NoResults from 'my-sites/no-results';
+import QueryHelpLinks from 'components/data/query-help-links';
 import SearchCard from 'components/search-card';
-import CompactCard from 'components/card/compact';
-import { recordTracksEvent } from 'state/analytics/actions';
 import { getForumUrl } from 'lib/i18n-utils';
+import { recordTracksEvent } from 'state/analytics/actions';
 
 export class HelpSearch extends React.PureComponent {
-	static displayName = 'HelpSearch';
-
 	state = {
-		helpLinks: {},
 		searchQuery: '',
 	};
 
-	componentDidMount() {
-		HelpSearchStore.on( 'change', this.refreshHelpLinks );
-	}
-
-	componentWillUnmount() {
-		HelpSearchStore.removeListener( 'change', this.refreshHelpLinks );
-	}
-
-	refreshHelpLinks = () => this.setState( { helpLinks: HelpSearchStore.getHelpLinks() } );
-
 	onSearch = searchQuery => {
-		this.setState( { helpLinks: {}, searchQuery: searchQuery } );
-		this.props.fetchSearchResults( searchQuery );
+		this.setState( {
+			searchQuery,
+		} );
+		recordTracksEvent( 'calypso_help_search', { query: searchQuery } );
```

Comment:
This isn't being fired, it's calling the action creator directly, not the prop 😛

Reply:
Fixed in 7c489d2bf1c7ceb11a596eac21385f60b1bfc711

Reply:
🤦‍♂️ second facepalm for the day. Thanks!